OrbitGl/TimeGraphLayout.cpp

```diff
@@ -80,14 +81,24 @@ bool TimeGraphLayout::DrawProperties() {
   FLOAT_SLIDER_MIN_MAX(m_TextZ, -1.f, 1.f);
   FLOAT_SLIDER_MIN_MAX(m_TrackZ, -1.f, 1.f);
   FLOAT_SLIDER(m_ToolbarIconHeight);
+  FLOAT_SLIDER(scale_);
   ImGui::Checkbox("DrawTrackBackground", &m_DrawTrackBackground);
   ImGui::End();
 
   return needs_redraw;
 }
 
+float TimeGraphLayout::GetRoundingRadius() const {
+  float max_radius = m_RoundingRadius * scale_;
+  // The radius has to be smaller than certain values to draw a proper circle
+  max_radius = std::min(max_radius, GetTrackTabHeight() / 3);
+  max_radius = std::min(max_radius, GetTrackTabWidth() / 3);
```

Comment:
Why 3?

Reply:
The rounding radius is already clamped to half of layout.GetTrackTabHeight() (see Track.cpp, around line 126).

Reply:
Actually, I'd remove the clamping here, and add this to Track.cpp:

float half_label_width = 0.5f * label_width; <- add this
...
float radius = std::min(layout.GetRoundingRadius(), half_label_height);  <- this line already exists
radius = std::min(radius, half_label_width); <- add this

Reply:
Thanks! I'm not being able to reproduce again situations where 0.5 (or divided by 2) is not enough. But I used to have. Maybe when moving to track we also solve a bug.

Anyway, about erasing the next line: 
max_radius = std::min(max_radius, GetEventTrackHeight());

Am I being too perfectionist when I'm being worried about this kind of situations? (the blue line is outside the rounding radius)
![Screen Shot 2020-07-31 at 12 26 04](https://user-images.githubusercontent.com/8610429/89050936-d6daba00-d329-11ea-9d49-28cfec4064cd.png)

Reply:
This is an issue with the z-ordering, the rounded corners should hide the lines.  The rounded corners are actually a mask, the problem here is that the lines are drawn on top of it.